air/src/proof.rs

```diff
@@ -118,7 +118,7 @@ impl ProofOptions {
 
     /// Creates a new preset instance of [ProofOptions] targeting 96-bit security level.
     pub fn with_96_bit_security() -> Self {
-        let options = WinterProofOptions::new(27, 8, 16, FieldExtension::Quadratic, 8, 256);
+        let options = WinterProofOptions::new(27, 8, 16, FieldExtension::Quadratic, 8, 255);
```

Comment:
Is it that now we're taking 255 -bits of output from hash function ?

Reply:
This is actually the parameter for max remainder degree. In our context, this implies that the maximum size of remainder evaluations would be 2048 (because we use blowup factor of 8).